tests/functional_tests/ffront_tests/test_execution.py

```diff
@@ -870,6 +870,43 @@ def solve_tridiag(
     np.allclose(expected, out)
 
 
+def test_docstring():
+    size = 10
+    a = np_as_located_field(IDim)(np.ones((size,)))
+
+    @field_operator
+    def fieldop_with_docstring(a: Field[[IDim], float64]) -> Field[[IDim], float64]:
+        """My docstring."""
+        return a
+
+    @program
+    def test_docstring(a: Field[[IDim], float64]) -> Field[[IDim], float64]:
+        """My docstring."""
+        fieldop_with_docstring(a, out=a)
+
+    test_docstring(a, offset_provider={})
+
+
+def test_domain(fieldview_backend):
+    size = 10
+    a = np_as_located_field(IDim, JDim)(np.ones((size, size)))
+
+    @field_operator(backend=fieldview_backend)
+    def fieldop_domain(a: Field[[IDim, JDim], float64]) -> Field[[IDim, JDim], float64]:
+        return a + a
+
+    @program
+    def program_domain(a: Field[[IDim, JDim], float64]) -> Field[[IDim, JDim], float64]:
+        fieldop_domain(a, out=a, field_domain={"IDim": (1, 9), "JDim": (4, 6)})
```

Comment:
Was there a reason for `field_domain` instead of just `domain`?

Reply:
"domain" is used in other parts of the code, e.g. when defining the size of the output field in the embedded. I used "field_domain" to avoid confusion but I can change it back

Reply:
In this other case it refers to the same concept: it defines the domain where the computation is applied. I'd propose to make it `domain` here.